src/internal.c

```diff
@@ -9493,6 +9493,8 @@ static int DoVerifyCallback(WOLFSSL* ssl, int ret, ProcPeerCertArgs* args)
     #endif
 
     #if defined(OPENSSL_EXTRA) || defined(HAVE_WEBSERVER)
+        store->depth = args->count;
```

Comment:
These defines don’t match the ones in the internal.h for depth. Can you double check those they should match?